libs/platform/experience/src/color/color.model.ts

```diff
@@ -0,0 +1,25 @@
+import { ColorId } from '@spryker-oryx/utilities';
+
+export const enum ColorTone {
```

Comment:
I'm not so sure about this semantic list. These are quite opinionated and none of our compnents have implemented their designs according to radix semantics. Can we move this to a numbered list for now?

Also, canvast is likely not correct. We're using the additional 0-color for the text color.